src/patterns/task-list-pages/index.md.njk

```diff
@@ -75,6 +75,32 @@ For example, you could group all tasks which help users find out if your service
 
 Where possible, allow users to complete tasks in any order. This will help them plan their time and complete sections as and when they can.
 
+### Marking tasks as completed
+
+Give the user control over whether a task is completed or not when:
+
+* some of the questions are optional
+* there is a textarea which may take longer to complete
+* the answer requires a list of things such as previous jobs
+* answers need checking carefully with a third party
+
+In these cases, ask the user whether they have completed the task, or need to come back to it later:
+
+![A screenshot showing a question that says 'Have you completed this section?'](have-you-completed-this-section.png)
```

Comment:
Can we put the radio labels in the alt-text as well, so that users who can't read the image can get the same information?